test/pylib/manager_client.py

```diff
@@ -190,3 +199,15 @@ async def host_is_down():
             return True if server_id in down_endpoints else None
 
         return await wait_for(host_is_down, deadline or (time() + 30))
+
+    async def get_host_ip(self, server_id: ServerNum) -> IPAddress:
+        """Get host IP Address"""
+        try:
+            server_ip = await self.client.get_text(f"/cluster/host-ip/{server_id}")
+        except Exception as exc:
+            raise Exception(f"Failed to get host IP address for server {server_id}") from exc
+        return IPAddress(server_ip)
+
+    async def get_host_id(self, server_id: ServerNum) -> HostID:
+        """Get host id of a server"""
+        return HostID(await self.client.get_text(f"/cluster/host-id/{server_id}"))
```

Comment:
The previous version of this PR had `try`+`except`+reraise with additional message.
This was consistent with the function right above it, which still does that.

The code should be written in a consistent manner. The two functions have nearly identical logic, so they should be structured the same way.

I don't get why you decided to restructure it in this version. You did not give any motivation for it in your comments, so I'm guessing the change is unnecessary. It only prolongs the review process. I think we both want this review to end and the PR to get merged, so why?

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/88d45d24e55c1ac231e596be0fa3284984b9a858